src/datastore/postgres/schema/v5.ts

```diff
@@ -0,0 +1,26 @@
+import { IDatabase } from "pg-promise";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create schema
+    await db.none(`
+    CREATE TABLE metrics_users (
+        user_id TEXT NOT NULL,
+        remote BOOLEAN,
+        puppeted BOOLEAN
```

Comment:
again, might be best to link against the users table and infer if the user is puppeted perhaps?

Reply:
One of my reasons for keeping the metrics separate is, if we had a constraint to functional data, deleting a user or room would reduce the team's usage of a system.

Reply:
I removed the metrics_users table.